classes/class.pmproemail.php

```diff
@@ -264,13 +297,12 @@ function sendCheckoutEmail($user = NULL, $invoice = NULL)
 			$this->data = array(
 								"subject" => $this->subject, 
 								"name" => $user->display_name, 
-								"display_name" => $user->display_name,
 								"user_login" => $user->user_login,
 								"sitename" => get_option("blogname"),
 								"siteemail" => pmpro_getOption("from_email"),
 								"membership_id" => $user->membership_level->id,
 								"membership_level_name" => $user->membership_level->name,
-								"membership_level_confirmation_message" => $confirmation_message,
+								"membership_level_confirmation_message" => wpautop( $confirmation_message ),
```

Comment:
We may instead want to use the `pmpro_confirmation_message` filter which will `wpautop()` the message and let add ons filter it. Here is how we use that filter to display the confirmation message on the confirmation page:
https://github.com/strangerstudios/paid-memberships-pro/blob/1cbc4daf1a8977d2c9d9887ec84adac83c02a0c1/pages/confirmation.php#L48-L54

```suggestion
								"membership_level_confirmation_message" => apply_filters( 'pmpro_confirmation_message', $confirmation_message, $invoice ),
```